Makefile

```
@@ -23,6 +21,23 @@ CROSS_BUILD_BINDIR ?=$(OUTPUT_DIR)/bin
 PREFIX ?=  /usr/local
 DESTDIR ?= $(PREFIX)/bin
 
+# Autodetect $ENGINE if not set (podman first, then docker)
+ifeq ($(origin ENGINE), undefined)
+  ENGINE = podman
+  ifeq ($(shell which $(ENGINE) 2>/dev/null), )
+    ENGINE = docker
+  endif
+endif
+
+# Container-engine-specific options:
+ifeq ($(ENGINE), docker)
+  CONTAINER_MOUNTOPT=
+  CONTAINER_SOCKETOPT=-v /var/run/docker.sock:/var/run/docker.sock
```

I think it needs quotes here
```suggestion
  CONTAINER_SOCKETOPT="-v /var/run/docker.sock:/var/run/docker.sock"
```